### ISSUE TITLE
Add publisher/journal: Federation of American Societies for Experimental Biology (FASEB)

### ISSUE THREAD

**Issue by mariaguerreiro** [CDL-Dryad/dryad-product-roadmap#2986]
Please note that their membership formally starts on January 2024.

**Name to appear on membership page:**
Federation of American Societies for Experimental Biology

**Payment plan:**
Membership fee covered by FASEB with DPCs covered by Wiley

**Journals included:**
_The FASEB Journal_
_FASEB BioAdvances_

**Email contacts to receive submission notices: (note whether they receive all notices, or different lists for different journals)** 
https://docs.google.com/spreadsheets/d/1NfxJv8m4PdBmRMqo0cH79tZjUa5Qfj7OzA0SvnG2_dg/edit#gid=1888868475

**Email contacts to receive publication notices: (note whether they receive all notices, or different lists for different journals)** 
https://docs.google.com/spreadsheets/d/1NfxJv8m4PdBmRMqo0cH79tZjUa5Qfj7OzA0SvnG2_dg/edit#gid=1888868475

**ORCIDs of users who need access to admin dashboard: (they must log in to Dryad to set up their account before we can give them access, note whether each user administers a subset of journals)**
https://docs.google.com/spreadsheets/d/1NfxJv8m4PdBmRMqo0cH79tZjUa5Qfj7OzA0SvnG2_dg/edit#gid=1888868475

**Email contacts to discuss integration of journal metadata with Dryad:**
sbhargava@faseb.org

Development tasks:
- [x] add member name to membership page
- [x] update payment settings
- [x] update journal notification settings
- [x] add journal administrators
- [x] start integration process (track on new ticket)

**Comment by mariaguerreiro** [comment]
Related to this, FASEB will use the API eJP integration. eJP asked them to reach out to us for *an individual client key and secret for this implementation*.

Please can you also facilitate these details?

You can also email them directly to eJP if you prefer. You can email the details to Jason Sklenar at eJP [support@ejpress.com](mailto:support@ejpress.com). You can mention eJP ticket # CT-2023-10-24-41169 in the email and in the subject line.

Thanks!

**Comment by mariaguerreiro** [comment]
@ryscher FASEB chased me about the individual client key and secret for this implementation.

Can you facilitate this information, please, or email them to eJP directly with me in copy? Thank you!

**Comment by mariaguerreiro** [comment]
@ryscher Can you let me know if we've sent the API key to eJP? FASEB is waiting on it to turn on the integration, thanks!

**Comment by ryscher** [comment]
Sorry about that @mariaguerreiro -- I had sent a number of API keys last week, and I thought this was one of them, but it looks like I missed this one. It is now sent.

**Comment by mariaguerreiro** [comment]
Thank you so much!

**Comment by ryscher** [comment]
- added FASEB organization under the Wiley sponsorship
- updated settings for the existing journals based on the provided spreadsheet
- created ticket #3027 to track integration process
- waiting for admin contacts to login before giving them access to the dashboard

**Comment by ryscher** [comment]
I haven't heard from the admin contacts, so I will close this ticket. I see that one of them did an initial login, so I gave them admin access. The other users can be updated in a separate ticket.

**Comment by mariaguerreiro** [comment]
Thanks @ryscher When will the new member be listed on the website? I do not see them there yet. Thank you!
